interface/forms/clinical_notes/templates/report.html.twig

```diff
@@ -0,0 +1,41 @@
+{#
+The Report template of the clinical note form.
+
+@package OpenEMR
+@subpackage Forms
+@author Robert Down <robertdown@live.com
+@copyright Copyright (c) 2023 Robert Down <robertdown@live.com>
+@copyright Copyright (c) 2023 Providence Healthtech
+@license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+#}
+<div class="w-100">
+    {% for n in notes %}
+        <div class="d-flex justify-content-between bg-light border border-dark p-1">
+            <div class="form-group mb-0">
+                <div class="font-weight-bold">{{ "Date"|xlt }}</div>
+                <div>{{ n.date|date() }}</div>
+            </div>
+            <div class="form-group mb-0">
+                <div class="font-weight-bold">{{ "Type"|xlt }}</div>
+                <div>{{ n.codetext|default("Unspecified"|xl)|text }}</div>
+            </div>
+            <div class="form-group mb-0">
+                <div class="font-weight-bold">{{ "Category"|xlt }}</div>
+                <div>{{ n.clinical_notes_category|default("Unspecified"|xl)|text }}</div>
```

Comment:
above line is not matching `category_title` that was used (`<?php echo text($value['category_title']); ?>`)

Reply:
Fixed